site/content/en/docs/manual/basics/attach-cloud-storage.md

```diff
@@ -215,6 +276,32 @@ button on the `Cloud storages` page and fill out the following form:
           - `Account name` - storage account name.
 
     </br>
+
+  - [**Google Cloud**](https://cloud.google.com/docs):
+
+    - [`Bucket name`](https://cloud.google.com/storage/docs/creating-buckets) - cloud storage bucket name,
+      you can find the created bucket on the [storage browser page](https://console.cloud.google.com/storage/browser).
+
+    - `Authorization type`:
+
+      - [`Key file`](#create-a-service-account-and-key-file) - you can drag a key file to the area `attach a file`
+        or click on the area to select the key file through the explorer. If the environment with the CVAT instance
+        deployed is specify `GOOGLE_APPLICATION_CREDENTIALS` environment variable, then it will be used if you do not
```

Comment:
```suggestion
        or click on the area to select the key file through the explorer. If the environment variable `GOOGLE_APPLICATION_CREDENTIALS` is specified for an environment with a deployed CVAT instance, then it will be used if you do not
```